Docs/LinuxOsxInstall.md

```diff
@@ -13,15 +13,21 @@
 - Make sure you have OpenGL availible
 
 ## Linux installation instructions
+### Arch 
+The AUR package `factorio-yafc-git` points to the original upstream project by ShadowTheAge. Download the PKGBUILD and edit the url to the following: https://github.com/have-fun-was-taken/yafc-ce
```

Comment:
I would add the link to the AUR package (https://aur.archlinux.org/packages/factorio-yafc-git)

Reply:
Done in 6c1b720 thanks!